eng/native/gen-buildsys.cmd

```diff
@@ -52,16 +52,10 @@ if /i "%__Arch%" == "wasm" (
 
             set "EMSDK_PATH=%__repoRoot%\src\mono\browser\emsdk"
         )
-        :: replace backslash with forward slash and append last slash
-        set "EMSDK_PATH=!EMSDK_PATH:\=/!"
-        if not "!EMSDK_PATH:~-1!" == "/" set "EMSDK_PATH=!EMSDK_PATH!/"
-
-        set __ExtraCmakeParams=%__ExtraCmakeParams% "-DCMAKE_TOOLCHAIN_FILE=!EMSDK_PATH!/emscripten/cmake/Modules/Platform/Emscripten.cmake"
-        set __UseEmcmake=1
+        set EMSDK_QUIET=1 && call !EMSDK_PATH!/emsdk_env
```

Comment:
Any opinions on `EMSDK_QUIET`? Windows was "quiet", not-Windows was "shouty", I made them both "quiet" because `emsdk_env` prints the full `PATH`. But I can very easily be convinced it's better to be more verbose with this thing.